config.toml.example

```diff
@@ -306,9 +306,15 @@
 #
 #debug = false
 
+# This will make your build more parallel; it costs a bit of runtime
+# performance perhaps (less inlining) but it's worth it.
+#
 # Number of codegen units to use for each compiler invocation. A value of 0
 # means "the number of cores on this machine", and 1+ is passed through to the
 # compiler.
+#
+# Defaults: the Cargo defaults, 256 with incremental and 16 without
+# https://doc.rust-lang.org/cargo/reference/profiles.html#codegen-units
 #codegen-units = 1
```

Comment:
So what I was suggesting is you could actually write this (we do this elsewhere in this file):
```toml
#codegen-units = if incremental { 256 } else { 16 }
```